### PR TITLE
lmp: add webhook notification for PR jobs

### PR DESCRIPTION
**Pull request by mwasilew**

### CODE REVIEW

File: lmp/jobserv.yml [hunk 1]
```diff
@@ -356,6 +356,10 @@ triggers:
 
   - name: Code Review
     type: github_pr
+    webhooks:
+      - url: https://conductor.infra.foundries.io/api/lmp/
+        only_failures: false
+        secret_name: lava-webhook-key
```
Comment (MrCry0) [inline lmp/jobserv.yml:362]: I'd use a more random secret name.

Comment (mwasilew) [reply lmp/jobserv.yml:362]: it's the name of the variable. Actual secret is not shared in this file.

**Review by MrCry0 — APPROVED** [review]

**Review by MrCry0 — APPROVED** [review]
LGTM, I'd just extend the commit message, adding details of what this hook is used for.

**Comment by mwasilew** [comment]
np, I'll do that in a moment

**Comment by mwasilew** [comment]
Commit message updated

**Review by quaresmajose — APPROVED** [review]

**Comment by mwasilew** [comment]
I don't have enough privileges to merge this.

**Comment by MrCry0** [comment]
> I don't have enough privileges to merge this.

Added @doanac and @ricardosalveti - they have permissions.

**Review by ricardosalveti — APPROVED** [review]
LGTM

**Pull request merged**